apiserver/facades/client/charms/client.go

```diff
@@ -154,40 +154,6 @@ func NewCharmsAPI(
 	}, nil
 }
 
-// CharmInfo returns information about the requested charm.
```

Comment:
I thought this was still used then I saw we had a common func so good to see this go

Reply:
> Did you test scp also?

I'll do that before I land